src/src/style.css

```diff
@@ -0,0 +1,79 @@
+:root {
+  --default-color-link: #000;
+  --color-active-link: #00acdc;
+  --background-header: #fff;
+  --box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  background: var(--background-header);
+  box-shadow: var(--box-shadow);
+}
+
+.logo-link {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  display: flex;
+  margin-right: 20px;
+}
+
+.nav-item:last-child {
+  margin-right: 0;
+}
+
+.nav-link {
+  padding: 23px 0;
+  font-size: 12px;
+  color: var(--default-color-link);
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav-link:hover {
+  color: var(--color-active-link);
+}
+
+.is-active {
+  position: relative;
+  color: var(--color-active-link);
+}
+
+.is-active::after {
+  position: absolute;
+  height: 4px;
+  width: 100%;
+  flex-shrink: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
```

Comment:
The `left: 0;` and `right: 0;` properties are not necessary if you've already set the width of the element to 100%. You can safely remove these properties.